Source/SwiftyJSON/SwiftyJSON.swift

```diff
@@ -1212,7 +1212,7 @@ public func > (lhs: JSON, rhs: JSON) -> Bool {
 public func < (lhs: JSON, rhs: JSON) -> Bool {
 
     switch (lhs.type, rhs.type) {
-    case (.number, .number): return lhs.rawNumber < rhs.rawNumber
+    case (.number, .number): return lhs.rawNumber as NSNumber > rhs.rawNumber as NSNumber
```

Comment:
@deathdisco Shouldn't this be `<` (instead of `>`)?
(Should explain for the failing tests 😉 )